### PR TITLE
Remove individual image-based lighting parameters

### LINKED ISSUE #10232: Remove individual image-based lighting parameters in 1.94
The `ImageBasedLighting` class introduced in #10226 deprecates the individual parameters related to image-based lighting in `Model`, `ModelInstanceCollection`, and `Cesium3DTileset`. These options include:

- `imageBasedLightingFactor`
- `luminanceAtZenith`
- `sphericalHarmonicCoefficients`
- `specularEnvironmentMaps`

They appear in the constructors of the aforementioned classes and in `Model.fromGltf`, and should be removed in Cesium 1.94.

### PR DESCRIPTION
**Pull request by j9liu**
Will close #10232 . Individual image-based lighting parameters were deprecated and are to be removed in Cesium 1.94. This PR removes those from the constructor options in `Model` and `Cesium3DTileset.` It also updates the image-based lighting sandcastle and updates `ModelVisualizer`, which is relevant for creating models with the `Entity` class.

To test the entity changes, I added a field to [this sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=xVl5b9s2FP8qhPdHFECR7TTZOtfJFqfdGqBZt6btjroYaIm2iVCkQFI5VuS77/HQrR5OnS1oEPF67/fuRzYWXGl0Rck1kegIcXKNTomieRq9tXPBfBDb8angGlNO5HwQog9zjhDlSzETNxO0xEyR0EwpwkisqeBnPKEx1kI2Vtc4EddqgrTMixmRs+SE0xRrUs7f7T6Z8zlnRCPCNdW3drjMuSWNVkSfCibkjBGenIuEBHFjuOvgSaJzyQtpmifeNU9EWrzJMiJPsSLB7ntgd9fL0TH6BackRJhla+xZxVaJdhF0WOf4rjzR5dELMVpKkZ4Y2o5ZiDIsFfmJCawDx3O3wDccotdrYo1npGCgPxxfKqRhUmnQKBJLJHKJUsopwlnGjE1AoGjO49Lu7uhRIQkzNpsPXpEEDF1OgdHezQe/r6kmxv7FOnz8LAnh7nPGcr/6J2FMXBcb8O188N7SsgJM0DgaVaRLKxi2z+lqzeBX15mXOxyK+h6LJGM49ozP6U3Bqzp6koqc6wkaRYe9K884XjCSNFyVMnBNojU57WqktehQ3U8hFamTpmqqhQv6D6hm387fPfF2h2i8IlJbU1dGTEm6IFJBZGqBLrmIL0WukVgoIq+MiAoM752tWI2szwQljd2Cw4zypEUeiJqJpy/PEcR5CrGpjIeZuTdn8AdDDGDG0BLCgOrSybQQbIFNZCQizs2xCALqmaMwuz1LIMf4PfOB4d+GaDz31uChfKUqqGFB2WFunTJKNHxeltLXT0JWM6YDfnafyhcqlnRBgjLmA0iFbzEY0Mc48rkoSg2FqAj2MjUU28NKYZGLVxvqd/cCaQlsEWQFzSeX8vxXgGzG6EZoa4mzJFDDW2X4Js4+MevnmxP+SEv2bvyjI7cRdQAdHfWXkej87I+tKM7h+EpD1yiBDmplYxtGbuW8+0NtEap7Z6H+MpSKicp0rXzpNmxHrpOvi7UvkKtPDLsjbEu+TblM/diGWIbOF/hVo3GKJYE2xIIKcgnA1sQUbs/KdZ2R5UehOEmSiiuwAwscqSI/ZEJRS64KQyw1fGH+yPZLT8kKiq7yit4b7z+KRt8dHHw7/t5r9uAgGh2OHn03+tZPOBzm24F3jNYEmxpT8TnHeg2N2yuYxlwF40eH9e0Z1fEaNo9qc1Iw1ppaZ63O+rlj86s5/gr2B55v6CiGlkidkZAUlISbOngtARKU2lRF6xbB33LQuuSw3auk0GAhfSYL0c1fZ3Cg3DYITpLAOwWHFnaCjA2bFCdt2jWsk/rAL1uvmhSuhoAirZNFtldN8/RXekOYa33G+4+rVXxjVi+gzzBN0Qh+wkbWnnyi2LULc9if7yc99aeveNi7Rg+Not2sRcrHa09FoNNydvLHl2SQzyZLMHsPS6fpXsTNTcXhu7CM+TKQbStJkmeFO5WXN3tb8a6cGWdQsPrOnPR+oMmNNj32CZWxxMui+wdv4u5KCb15mbB2K++p55dKA/NBFA3h3wVOM0aeYo2H1u3U0IUOcPFff8NntGKLkp/5OQSnikaltp40BA57UP8swZIJekvWNGbkobA7Lp5Jc9QRYRP0z4VGoAY0g9ZdCP6wqvdMmqMO/PGmFjin7BK9lnl8+bDwDR/Lpj3+KgtcXFLOof88XWOIIG2fWB5SCFzo/2/43AT5e1swLjBPYqw0+B0UiNfuDnZOeB744N7t3bZaMTLLtTZFaXDhnoLMhdgnD/84FNYktQVwTUxOmfNmz+B3m17fbXhSPht9ku/Z7IXhaR+aPsOJLgM/1d8Z0RSvyAwrkrwwvQTU35/AeKJV68tOZT8wV3tzv/ftEtyhFdmUcodq9NezVy8dwVIDcz4IB1Olbxk5Loz5I00zIbUptQH4hibgG+A7argA7yU6ipUq7I3QN8WV/UPlFgvI7SubdCZIrhY4ONgPUfE7ih5Xp6E5AMUD6gk6yG5q0wshEyL3JDQruWou3nVYU57lug7AvHdQqPt7mNEV9BYpTRJGulz3tMigNWhwLpYWAvwgba52Wdt+ijSkXwqu965t0zgBOVjSPj4d1vU9TegVoslRz+MpihlWClaWOWOuNz+eDmF/5yjUYQP6JQjOzIvR8XQ9Pn7hJqMomg5h2H+yfE0pzT/V5opwXAk01QuR3NYmzJRsjM1MUsJ1OgGS7i3Idh3ToU6aJIYtGr00LYnOWbfUnIE5lwBRAmlsb0G5Ec+nmUn7dRByibmOtOetgh2ZNsN74u8X3q19HnALZx3efQHZ/u5LNeoiS99mBKDBJWIFGjJtN4xM1R2YJhu+x/ZbaZK5hTGM6kJ5GWwr7SV6kyX2HX/Hstip+V8/c1MADQ+IAhgdfoJBh9Q9NdVgYCMW7iLFzeFj70I/oJ0dBFKtJL7dueuRyrw6fxrdf2KKNv5+q4RQbIxcH5V3O2brgtmU7X1N/JGMdVHeYx4i7DfKW+3/QCgTQueVbduZ639MFK3b6EOmjM6j3pa0Z8r1tpU32lR7BsR/ozzfnWymOxg2OgsY11qPer/yLw) to toggle the image based lighting factor. Seems to be the only IBL-related parameter to apply to models created with the `Entity` class. Maybe we should add the others in at a later time?

EDIT: I am not sure why Github isn't showing it under files changed, but I edited `Source/DataSources/ModelVisualizer.js` for this PR.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @j9liu!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/remove-ibl-parameters/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Source/Scene/Cesium3DTileset.js [hunk 1]
```diff
@@ -754,8 +731,9 @@ function Cesium3DTileset(options) {
   /**
    * The light color when shading models. When <code>undefined</code> the scene's light color is used instead.
    * <p>
-   * For example, disabling additional light sources by setting <code>model.imageBasedLighting.imageBasedLightingFactor = new Cartesian2(0.0, 0.0)</code> will make the
-   * model much darker. Here, increasing the intensity of the light source will make the model brighter.
+   * For example, disabling additional light sources by setting
+   * <code>tileset.imageBasedLighting.imageBasedLighting.imageBasedLightingFactor = new Cartesian2(0.0, 0.0)</code>
```
Comment (ptrgags) [inline Source/Scene/Cesium3DTileset.js:735]:
```suggestion
   * <code>tileset.imageBasedLighting.imageBasedLightingFactor = new Cartesian2(0.0, 0.0)</code>
```

File: Apps/Sandcastle/gallery/Image-Based Lighting.html [hunk 2]
```diff
@@ -145,6 +145,8 @@
           })
         );
 
+        const ibl = model.imageBasedLighting;
```
Comment (ptrgags) [inline Apps/Sandcastle/gallery/Image-Based Lighting.html:148]: If this is only used inside the callback, might as well put it just before the first use, if only for easier readability.

File: Source/Scene/Model.js [hunk 3]
```diff
@@ -1232,87 +1208,6 @@ Object.defineProperties(Model.prototype, {
     },
```
[the remark this answers is not in the document]
Comment (ptrgags) [reply Source/Scene/Model.js:1208]: Oh this change also needs to happen in `ModelExperimental.lightColor`'s documentation.

**Review by ptrgags — COMMENTED** [review]
@j9liu Looks good, had a few comments.

I also set up a [local Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=tVltb9s4Ev4rhD/JrSs7TrJ762R75zjpNthkN6i9BQ51UTASbROhKC9JJfYW+e83fJFEybITN7mgcCVy+Mxw+MxwSEUplwrdU/JABPoVcfKARkTSLAk/m7Zg2orM+yjlClNOxLTVQd+nHCG5SDMWDzlNsCIDpERGOlP+2D6Z8sigyohwAqAWPTSvRSf8KrJS0G3aQ/de9uOECFyOtu/QbcVjcpvNx4v04YOADnlDxJjAwBgGaENALgei8IN5ROKhEHgtL1aKcElTDpJOZ/itJgOjGdEmZlwbeNDrH7kmucQR5XNo7IW9Xq/vmjPBoGnaCsMu/BvjZMnIOVa4m6QxYbJrXTqkwj19g8dwzm6nrRxAklHKGImUtaw2B5iyIuIq5XOqslj79N3Px+FPB/3eL0eHByc1MaxyqSOw8qh/fNgD0UJqQeh8oed1DL1GySzjVnEqKECMjKsDi9ZBAsc0k2276BZCYD7X+NdYLcIErwIr03GrFs5EJhVwiBMs2ugN6htF+egFAWnjREc1A6PST9CMuQz6h2BY2xuwpCpabBV/1y/FrXqWpndDFegW5JzSsS8evz9aK2409ic9n8DZ1bH6OnaWbT2wAX0CvXKWiiQorRJ0dRRenl/8Mbmc/FePedTefaA8Th+A4N76ZjwmM4imuOr/SBCIpZIJQU7MqvsrUDYeloImVNF7IkMcx8HGbK81ES8dmqfgu5VEmsID/dPJGwrVg/LRdT56TmmaXflipoe8hhBmGK9vRJpQSSxaqBaEB4UPglLYTVr/dbvohuE1whDisABLhBmDF517QFAiDLTCbPZOLgmJ80GeWhxp5wyLAdpLQ8bK+SOUZEzRJaNEDCC6jztlj1Y4qHiyALqCrvDTxc3FcFIMeLSe0X+VgCoBPcO+3UKW0bQbLxdEQB706fqErI26XDTXalcH/BphoLHnWCJEKjyfuqXDjAjlOguInLw1aprJyy20XKaUKwnr/+WrwdGJzSTAggVaihE+VzqaC5DQNpkxEFEo0AOpzrEn8N+pG3GC3r6lhfXV3O6jfaFfT3wZY4ENTk8s9Jor4krHNcOOyV4EjbBQ8IT5YZC7qRb5c2Kzgh0deBo6Pmw+2rorXGZyEdS6rYAB2B3hqMJKyLtp8htTM5/WG4GNfJcM/BePwfapZktD8G6Gr5H0WPaDsZurfEbg7grdvYO3Er65H54RTLvCqcB8tO7cdOZuJ1pmVoO/Ss6zSl+doNVew5IbQ77AcrBTw86HN2av3fmqMUOht27KO/KXc9Bm4hFEEhU4X2yEgiBJCvQAQvgFQ5EMymRkO+aCxmP6T1G7yL9hrUydl1O9SEBrm4DWkIDyUToFrYtVKSRXVnJVk1x57HCpzyvg6iXd26K2fIOCFXpXmtpF/ZJIDqgs8Wo1XwVm/QTMMpXUJbqNJGdYck7mghAZ+FuhM9jfHp1yr8nWmP7OVNVcloBmGSD7xWkSbNjn6r5dMgK2xydEZAQxCTJBRQhcdQB1o/aLK4P9MdVdw3mnqPlkuPCrRzBhkn6gKxKb04jnrtzDnmu2laAAslmBQmO7vr9v7Dsu+63P1n9xqs0b6wkHjfneM8S4pXlXaFq4crs2O9b3vbaTvK7wkiCdoaBy7CnT3fYq2CYJBDUIqUpvFCZ5vjXaYAcyPgG/RnckLudSnImAFKx+Ah4DNOFjiCdycQ9h9tEKBcWJlN9jo8iNDiFRXfJlpobWbD+d35MEEKrlkrHlsixhXHKD5mJAmBPIzkf7zBnnzg9BFaVd2TNSRuC0Mq/LnGwpoSqrka/dv2t25u2DWnuZlCvwhmT1OMqJq1NMhbIH4UF7R/G2hfdBnaJhEwk9S4o2/4BkQYO2vx01Q4Il9epRE62zjTeT9ZKEVxcfJt9GV5ej321QjYEzEZYKlggqmkmaslssrgnPgi8a0K2ivp4YoGlLX0F4rG25GYCPiF6vASq5Vtl1KhcYttHtp3lEGiT7u6Gz9xKVvf019o9/XGH/eH99L3Dpj/nzhR59hk+//gi7hlREAs/Uc0160XWXi8bNi7Q93fmb0MUk+kwWNGLktUy3qA60+rZ7Br1/7T+Fa8ru0ERk0d1e5pc771NLoBUY/Pr7689lfEc57EdotMACzmzmnvgVyXSNeU4meHzC/IP/T5i4rU5ruuD4lsFh9dkx3HzR7F1I1K+niylsuRrec3k828+pfC3j/WvM17U2pvc0zjCzO+2zk2Vd/QxDpfgiNpxlSkEtB5vTuf7ysaHVFm1JKpaLSdo/D9xd9ONzIEefn4aE6WTJbSb155i9wA+fYe/hU/bO54wUiJdnVxrSfOgpgINoQXQd6OAbKtSNG+J27WJjBvki1bW3g4Kas+nerQ/lYa9jD22DLRI9LVF8Dmi4nQ5pgufkDEsSX+kjKgREQ9OH3CJrWlHiWSe1Oq1TqdaMvM+J9R+aLFNhvgMFkMQUgSQGhxLZvYVcS1QYSZlbdNr1h54CzxGNf234yoYihqWEnlnGmD7BT1vvT7sgvzGUpebU+Oc9EQyvtdji4P2VbQzD8LQLr80jleVMDfl/) for testing `ModelInstanceCollection` with IBL, that's working nicely.

One thing is in [this ModelExperimental sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=nVh7c9s2Ev8qOP0T6UbmAyRF0q+52k5S3zkXT+x05lp1HIiERNQkwQFB27qOv/stwLek1s1lMjK52Odvd4EFI56XEj0x+kwFOkM5fUaXtGRVZvykadPlJNLvlzyXhOVULCezk2W+zNkaTf9WCxplRHNqlAWNqpSI9/kTEzzPaC4/kaK8q4qCC0njGfp9mSP0zPKYPxskpUJOFQGh5eQ+YSVaCf5cgh8xpyXKuURlLYpazYj2qlEGuo3lRGlQHr0qpyIdDh058PXLDTpTXMtJImVRHptmHZIR8cwsqlXKIvOO5HFESpnSO5IVKb0ikpiPjAjysHgga0lFznn+gB8f2Co1HuULXk5OWnsZj2mqzYANwzDh/0CLXi3NW/KcN7/GJl1p8WVumkiHvhMYeiYlKgQHR0sao6pk+Qb9KwEOXr4r0Sa9/4CuL25QbUYY6J637GjLK4H4c36stdvoCF3BW8pJjCBItKpYGiOZ0FbfnjY0bXHaMJlUK41Tw/xR8KowlcQRSBw1EjND28Jg60uVo29RytARy4tK3hKZILOAX1Nyk2VkQ40kFuiIV/KyWlHIT78MNBDR6H4zBpqSb2jNBSJpinghGWButNhxtIHSE0RSHVJZJFSwiKQoISLjOYtQxOl6zSIGyEKF0ZQ/zwFPij5yvkkpgPmBpUTjDgD+RiN5ADelumMTNKUEFBxCaaOVmuuG12x4yz2AMnAbHR3JbUHPIF54jGmR8u3ZDhgHsANoPoNDQmGSEQiKCArYlLxtFxobYysJTYtDEIJHDnh0Tx5r9Foo4zFoqmpaGDLE8maH+OcdgiItEwCpxtXoO/DGssa7ySUBv0pGcke3vG1gx/WtcIEt1/f8RThvqNh2FzYOF1Zge67TUn3H9hZBYDsOdkJ/mc+61ruxH+w3LNmOF2LbxtbCdRb2otFp+za2fT8M7WDhW35rKfAXruu7TmiHju+MLb0ZkuspKTsMFk4Q4nlH9cCIHwShBdY6QzgAYuhiz3KDYGzozyM6AuetRehYvoN97FterVORbexh7IJqH1tW2JE9B3wKnAA7TuiObeEH/JYxK1hgvAhdy/Y8C3daLT+EtISujbFju15nzLJDG7thACgCwjvG3sxVEAZQGLYTOl7gtRDagRdakH+I1XX7qrBdB3h82/KxswMh/vNcWYYFuQ8s14NSWzTVB0Rs2ZB1z9X11hEBPsdbhNiBpO0EZP8F8AIAD1ueFTpO0IEXQJnb4APE43pBn0AoU9eC0sCh5++E9EaiwFMPu7YbuJAqq8FOUd3AscC0BT0FldiTbegyaAHLCxZeODQ1av4z9As081w3mvrVj+oJKkf/6mdFxfoJ/3rSbwMJZZtEgg6w16lPip1J40dKYjjfbpmMki88TafAPUftT+8XF2wDu8/ZPgLGWvDsim4EpWWDPHagQl1AuY7YdQEey/GtOq21Y8Og9Sn9iUjBXnoL94LkpdpmSyPZcfKef2AvNP4gYKPXJmvvau2FqFWPlIPa0bhUCJYxyZ5oaZA41joas58U+/uXAo4ydYqQVMf3MZXr6e/1wFSJ9LgbO+Y1bRDB8fClXWY5y6rsFrxO79h/6TGycV14r7OxtzDhgK9ag6FPnQs4wuIbhRggoBnrRQGYbG/BNVZSpciAAyKfrqs8UicMmmquZupDqCkuwEuQHov6XStVTHAk/cx5hiSvHRiJwggqAHo9qY4nzwgyn98VJKKXWt9lx3oy1CDqQVD9w4aF/o4AocTIyEvtqrHiVa6yfKfGCGoISHlVzhufIQlVKSE9OSVi1uttDBkNwMr9K1ZKkkcU/BRgxTK8LsAmFEirDqPNuM6Ua2RVKlmRbi+2t5zl7Xjc5NbYT2q7suN4rb7jObxrzOrl2QihpsxHniWG5F+ADL0wxU7Tkr1IoVriDwWO8FCgBjLl/PGHPrg/9na0M5B8Q6eNf/Pa6lzDC6ncjWVop2vi6Q7Y11fv/31/ff+fWZccNdt3I+SPzQR5Od4Oh7vjyVDs4OUHBPZuI8Niv4exS5WybnkkBYkey3qSlWqm5WukpnlVWogUBdxUiGotY4h/L37WthpCaQUyqgR/kD/THAbUY+3lHrlB/fWk8wha5wluZtqJXnVGsxUVJUx/0JmPOY8eYT5FfAXXtSeySmnZuNRA3HIYOqJpp2c2jP0CLoI7ZkC5Ilx9/oRgcs405ICBon29hj9EbSAww6pZlskRDJLzdEVUT8U8qpSosaHyfa3lYnsNO+yk4amvsIfcVRhvlV9QY2Xv9rzV3vvf7XJltYLth63oLVFnAVTyNIe/sz4XO0ZaMuyX4ODnDsKhOa1hwNgZGeyu0Cc/kbQaWmqK8Rcl/mt9yGqWk57htQ39tY3kQADLyV6htNd+JdFfltXJdc83cOe5qKTkedfTy8lXuCLpK2lcCbiPDS+4aXOULCdd06/hAkP7ty7IKKHRY/fpoIlQfXk4tPB/N/BQ+HvauAMVqhXC/W5XYNOma5bT+Dv8OCTz2j6+DvfBV11BcMLCTjmoGyoEFx1uo68x9VIjDH8m88lpKbcpPW8N/INl+osMDCBTwzAlhfs/bFSluaogHdKIyrItsFNzKHoasyfE4rMD35NQlJKyhJV1lerRZDk5PzWBf09U3UKhbj7DBpWSrWJL7PObmmgYxqkJr4clu87vIjmVqunOexBP5YrH2/NhJk6lGL0rSnx+0/YGgu2o7o5TE+j7nGMK0PQ3EaRv/cuJUCcahA/a4A2OSfVMXuAZ6+dS0qJesOEtJpIcrSBbQHpSTX28v8nPkV75WgAzrL/T5t4Ngj7siKQvUtkD6OHN/YvG9tTugQCEIX7wOkIY3gcpGObtfw) for IBL, the default camera view is way off, it's looking at space (I saw this before in #10379). I think that more likely has to do with bounding spheres rather than IBL.

File: Specs/DataSources/ModelVisualizerSpec.js [hunk 4]
```diff
@@ -176,7 +176,7 @@ describe(
       expect(primitive.clippingPlanes._planes[0].distance).toEqual(
```
[the remark this answers is not in the document]
Comment (j9liu) [reply Specs/DataSources/ModelVisualizerSpec.js:176]: I believe so, yes.

**Comment by j9liu** [comment]
@ptrgags Yeah I think there's a fundamental difference between `Model` and `ModelExperimental`'s bounding spheres. For model, we need to do this
```
const center = Cesium.Matrix4.multiplyByPoint(
      model.modelMatrix,
      model.boundingSphere.center,
      new Cesium.Cartesian3()
    );
```

Because `model.boundingSphere` doesn't have the model matrix applied to it. But for `ModelExperimental` this is fine:

```
const center = model.boundingSphere.center;
```

so... I'm not sure how to resolve this. Do we need to revert the `ModelExperimental` changes to be consistent with `Model`?

**Review by ptrgags — COMMENTED** [review]
@j9liu noticed one more place in the code that needs to be updated

**Comment by j9liu** [comment]
@ptrgags thanks for the catch, updated!

**Comment by ptrgags** [comment]
Looks good now, thanks @j9liu!

**Pull request merged**